config/initializers/modify_rails_form_builder.rb

```diff
@@ -0,0 +1,14 @@
+class ActionView::Helpers::FormBuilder
+  def error_message_for(field_name)
```

Comment:
I did think about it briefly but I was able to make the code happen without it fairly easily. I feel like for the functionality we need right now this is good but I could see perhaps down the road simple form becoming more appealing.

I also remember my past co-worker ranting how much he didn't like simple form so that also swayed my opinion.